package/src/components/SelectableItem/v1/SelectableItem.js

```diff
@@ -147,29 +210,61 @@ class SelectableItem extends Component {
     return this.state.value !== this.props.value;
   }
 
+  renderIcon() {
+    const { components } = this.props;
+    const { iconAmericanExpress, iconVisa, iconDiscover, iconMastercard } = components;
+    let icon;
+    if (this.props.item.icon === "iconAmericanExpress") {
+      icon = iconAmericanExpress;
+    } else if (this.props.item.icon === "iconVisa") {
+      icon = iconVisa;
+    } else if (this.props.item.icon === "iconDiscover") {
+      icon = iconDiscover;
+    } else if (this.props.item.icon === "iconMastercard") {
+      icon = iconMastercard;
+    }
+
+    return (
+      <span className="icon">
+        {icon}
+      </span>
+    );
```

Comment:
Each item should have the component itself on it, or rather the element (already rendered). That way the list component doesn't need to know what a credit card is. So you'd just do `<span>{this.props.item.icon}</span>`